File: 7class/Tema7/ex3.py
using namespace std;

int main() {
  int k, a;
    
  k=0;
  cin >> a;    
  while (a != 0) 
    {
        if (a>0 && a%2==0) k++;
        cin >> a;
    }  
  cout << k;    
}

File: 7class/Tema10/ex3.cpp
using namespace std;

int main() {

  int n, i, k;
  cin >> n;
  k=0;
  for (i=1; i<n; i++)
   {
      if (n%i==0) k=k+i;
   }
  if (k==n) cout << "Yes"; else cout << "No";
}